manifest.schema.yaml

```diff
@@ -98,16 +81,26 @@
         "filter": {
           "type": "string"
         },
+        "type": {
+          "type": "string",
+          "enum": [
+            "schedule",
+            "webhook",
+            "connection"
```

Comment:
Question: what's the default if I don't specify anything? Do I need to specify the type for connections? If I specify a schedule string without the schedule type is that an error?

Reply:
is there a test for these cases?